Client/src/index.css

```diff
@@ -1,4 +1,5 @@
 @import url("https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap");
+@import "https://unpkg.com/open-props/easings.min.css";
```

Comment:
What is this for?

Reply:
I think this one is responsible for importing the easing functions that is being used by the toggle switch. 

This are currently the easing variables that is being used:
- `--ease-elastic-3`
- `--ease-elastic-4`
- `--ease-out-5`

Reply:
@peterpardo we don't want the project to depened on a CDN that we can't guarantee is going to be available at all times.

Can you please copy only the css that is needed from that css file and add it locally?  We can drop the import then and ensure the code wil be working at all times.

Reply:
Got it. I'll check how they did the animation.

Reply:
Here is the new changes. I've removed the import and instead just took the code needed for the smooth animation of the toggle to work.

https://github.com/bluewave-labs/checkmate/pull/1409/commits/76dc4ebb0cfb8521ebf5780a8ae544957a758735